### PR TITLE
add dynamic fields

### PR DESCRIPTION
**Pull request by britt-mo**
## Description
Added dynamic fields (plan names and bssEntities) to the export page

### How to test
Create a program and add plan names and bssEntitities. Submit the program and hit the print button. You should see the names listed. Also works w/o adding any plans or bssEntities. In that case the fields  should be blank

## Code author checklist
- [ ] I have performed a self-review of my code
- [ ] I have added [thorough](https://bit.ly/3zPrxuZ) tests
- [ ] I have added analytics, if necessary
- [ ] I have updated the documentation, if necessary

## Reviewer checklist (two different people)
- [ ] I have done the deep review and verified the items in the above checklist are g2g
- [ ] I have done the lgtm review

### CODE REVIEW

File: services/ui-src/src/utils/other/parsing.ts [hunk 1]
```diff
@@ -73,3 +73,11 @@ export const parseFieldLabel = (labelObject: {
 export const parseFieldData = (data: string) => {
   return data || "";
 };
+
+export const parseDynamicFieldData = (data: any) => {
+  const dynamicValues = data?.map((item: any) => item?.name);
+  const formattedValues = dynamicValues
+    ?.map((value: string) => `<p>${value}</p>`)
+    .join(" ");
```
Comment (gmrabian) [inline services/ui-src/src/utils/other/parsing.ts:81]: Realized after the fact we could probably combine these, no? Use the map in the second one and change value to value.name 🤷

Comment (britt-mo) [reply services/ui-src/src/utils/other/parsing.ts]: Just updated

**Review by gmrabian — APPROVED** [review]
Looks good! Nice work

**Review by BearHanded — APPROVED** [review]
lgtm

**Pull request merged**